docs/varnish.md

```diff
@@ -0,0 +1,51 @@
+# Varnish
+
+Follow [installation guide](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/varnish)
+for instructions to collect logs and metrics from this application using Ops Agent.
+
+# `varnish` Metrics Receiver
+
+## Configuration
+
+Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your varnish instance configuration.
+
+To configure a receiver for your Varnish metrics, specify the following fields:
+
+| Field                 | Required | Default | Description                                                                                                                                                                 |
+|-----------------------|----------|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `type`                | required |         | Must be `varnish`.                                                                                                                                                          |
+| `collection_interval` | required |         | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`.                                                                                      |
```

Comment:
I was under the impression that this was optional and it defaulted to `60s`

See: https://github.com/GoogleCloudPlatform/ops-agent/blob/19e1eb67f06bf4497678df9a1f111cd1ca32f279/confgenerator/config.go#L393-L403

Reply:
[couchdb](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/docs/couchdb.md?plain=1#L18) and [rabbitmq](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/docs/rabbitmq.md?plain=1#L22) appear to have the required field too for collection interval. Not sure if you want me to make that change in this pr too or if that would belong in another pr.

Reply:
Separate PR is fine with me.